src/Logger.hs

```diff
@@ -20,7 +20,5 @@ data LogEntry =
   | LogEntryForModerator EventLogItemCold
   deriving (Eq, Show)
 
-type SendLogMsg = LogEntry -> IO ()
```

Comment:
* Documentary types, as we except only logging here.
 * More safe in case of refactor.
 * In my opinion converting type synonyms to newtype are better than removing it.